registrations/management/commands/verify_registration_schedule.py

```diff
@@ -0,0 +1,116 @@
+                  "next_sequence_number value. By default it will use "
+                  "datetime.now() (format YYYYMMDD)")
+        )
+        parser.add_argument(
+            '--sbm-url', dest='sbm_url', type=validate_and_return_url,
+            default=environ.get('STAGE_BASED_MESSAGING_URL'),
+            help=('The Stage Based Messaged Service to verify '
+                  'subscriptions at.'))
+        parser.add_argument(
+            '--sbm-token', dest='sbm_token', type=UUID,
+            default=environ.get('STAGE_BASED_MESSAGING_TOKEN'),
+            help=('The Authorization token for the SBM Service')
+        )
+
+    def handle(self, *args, **kwargs):
+        sbm_url = kwargs['sbm_url']
+        sbm_token = kwargs['sbm_token'].hex
+        registration = kwargs['registration']
+        message_set = kwargs['message_set']
+        apply_fix = kwargs['fix']
+        today = kwargs['today']
+
+        if not sbm_url:
+            raise CommandError(
+                'Please make sure either the STAGE_BASED_MESSAGING_URL '
+                'environment variable or --sbm-url is set.')
+
+        if not sbm_token:
+            raise CommandError(
+                'Please make sure either the STAGE_BASED_MESSAGING_TOKEN '
+                'environment variable or --sbm-token is set.')
+
+        client = StageBasedMessagingApiClient(sbm_token, sbm_url)
+        subscriptions = client.get_subscriptions({
+            'identity': registration.mother_id,
+        })
+
+        if subscriptions['count']:
+            raise CommandError(
+                'Subscriptions exist for %s.' % (registration,))
+
+        weeks_estimate = registration.estimate_current_preg_weeks(today=today)
+        voice_days, voice_times = registration.get_voice_days_and_times()
+        mother_short_name = utils.get_messageset_short_name(
+            registration.stage, message_set, registration.data["msg_type"],
+            weeks_estimate, voice_days, voice_times
+        )
+
+        message_set_info = utils.get_messageset_schedule_sequence(
+            mother_short_name, weeks_estimate)
+        messageset_id, schedule_id, next_sequence_number = message_set_info
+
+        sub_requests = registration.get_subscription_requests().filter(
+            messageset=messageset_id)
+        if not sub_requests.exists():
+            raise CommandError(
+                'No SubscriptionRequests exist for %s with messageset %s.' % (
+                    registration, message_set))
+
+        for request in sub_requests:
+            if (request.next_sequence_number != next_sequence_number):
+                self.log(
+                    '%s next_sequence_number is %s, should be %s' % (
+                        request.id.hex,
+                        request.next_sequence_number, next_sequence_number))
+                if apply_fix:
+                    request.next_sequence_number = next_sequence_number
```

Comment:
It won't affect anything currently with the few messagesets that can be subscribed to, but I would set the `messageset` and `schedule` here as well in case they have changed based on the recalculation. And though it is highly unlikely that the `messageset` or `schedule` would change but not the `next_sequence_number`, I'd check for that possibility as well in the `if` in line 99.